pkg/cmd/pr/create/regex_writer.go

```diff
@@ -0,0 +1,62 @@
+package create
+
+import (
+	"bufio"
+	"bytes"
+	"io"
+	"regexp"
+)
+
+func NewRegexWriter(out io.Writer, regexp *regexp.Regexp, repl string) io.Writer {
+	return &RegexWriter{out: out, regexp: *regexp, repl: repl}
+}
+
+type RegexWriter struct {
```

Comment:
Go uses 'regexp' in the standard library instead of the more common (in other languages) 'regex' - it usually makes sense to stick with the convention set there